src/modules/0.0.17/aws_secrets_manager/index.ts

```diff
@@ -20,37 +20,37 @@ class SecretMapper extends MapperBase<Secret> {
   module: AwsSecretsManagerModule;
   entity = Secret;
   equals = (a: Secret, b: Secret) =>
-    Object.is(a.description, b.description) && !a.value; // if password is set we need to update it,
+    Object.is(a.description, b.description) && Object.is(a.versionId, b.versionId) && !a.value; // if password is set we need to update it,
 
   secretsMapper(secret: SecretListEntry) {
     const out = new Secret();
     if (!secret.Name) return undefined;
     out.name = secret.Name;
     if (secret.Description) out.description = secret.Description;
+
+    // version id will be the AWSCURRENT one
+    out.versionId=undefined;
+    if (secret.SecretVersionsToStages) {
+      for (const [key, value] of Object.entries(secret.SecretVersionsToStages)) {
+        if (value[0] && value[0] === 'AWSCURRENT') {
+          out.versionId=key;
+          break;
+        }
+      }
+    }
     return out;
   }
 
   createSecret = crudBuilderFormat<
     SecretsManager,
     "createSecret",
-    string | undefined
+    CreateSecretCommandOutput | undefined
```

Comment:
I think this can be reduced to:

```suggestion
    CreateSecretCommandOutput
```